aait/backend/group-2B/Application/DTOs/PostDtos/Validators/DeletePostDtoValidator.cs

```diff
@@ -0,0 +1,37 @@
+using FluentValidation;
+using SocialSync.Application.Contracts.Persistence;
+
+namespace SocialSync.Application.DTOs.PostDtos.Validators;
+public class RemovePostDtoValidator : AbstractValidator<DeletePostDto>
+{
+    private readonly IUserRepository _userRepository;
+    private readonly IPostRepository _postRepository;
+    public RemovePostDtoValidator(IUserRepository userRepository, IPostRepository postRepository)
```

Comment:
Making the class name the same as the file name is better because it helps improve code readability and organization.

Reply:
Fixed